src/consts/windowMessages.ts

```diff
@@ -3,9 +3,15 @@ export const WINDOW_MESSAGES = {
   CONNECTED: 'FWC_CONNECTED',
   // WalletConnect Disconnect
   DISCONNECT: 'FWC_DISCONNECT',
+  // WalletConnect Session Update
+  SESSION_UPDATED: 'FWC_SESSION_UPDATED',
```

Comment:
This should match the method names, so this one is SWITCH_TO_GROUP

Reply:
Or just remove this and instead use the below SEND_WALLET_ACTION_COMPLETE/FAILED

Reply:
Changed my mind again, my vote is to have the window events mirror the methods exposed to the dApp, in this case, WALLET_ACTION is not exposed, instead we should have SWITCH_TO_GROUP_COMPLETE/FAILED